rootconf/default/etc/machine.conf

```diff
@@ -0,0 +1,16 @@
+# /etc/machine.conf for onie
+
+#  Copyright (C) 2017 Curt Brune <curt@cumulusnetworks.com>
+#
+#  SPDX-License-Identifier:     GPL-2.0
+
+# Source build-time machine configuration
+. /etc/machine-build.conf
+
+# Source run-time machine configuration if available
+[ -r /etc/machine-live.conf ] && . /etc/machine-live.conf
+
+# Use onie_machine if set, otherwise use build_machine
+onie_machine=${onie_machine:-$onie_build_machine}
+
+onie_platform="${onie_arch}-${onie_machine}-${onie_machine_rev}"
```

Comment:
I believe it is missing 'r' before `${onie_machine_rev}` .  Please see the original definition of `onie_platform`:

    RUNTIME_ONIE_PLATFORM     ?= $(ARCH)-$(RUNTIME_ONIE_MACHINE)-r$(MACHINE_REV)